### ISSUE TITLE
plugin.h Not Found?

### ISSUE THREAD

**Issue by summersmile1984** [MekkaSiekka/Face-Recognition-with-OpenVino-Toolkit#1]
Hi, the plugin.h file  does not exists, is it a file in the new version of vino?

**Comment by MekkaSiekka** [comment]
@summersmile1984 the plugin.h is only used for a WebRTC plugin. You can un-build it by commenting out the top of CMakelist  "add_subdirectory(*)" This you should not be worried.

**Comment by summersmile1984** [comment]
Thanks, seems it is another product of intel

**Comment by MekkaSiekka** [comment]
@summersmile1984  Yes. It is not required to be included for local recognizer. Feel free to ask if any help is needed. 

Chris.
